lib/cucumber/runtime/ast_tree_walker.js

```diff
@@ -196,6 +196,10 @@ function AstTreeWalker(features, supportCodeLibrary, listeners, strictMode) {
       return world;
     },
 
+    isDryRun: function isDryRun() {
+      return dryRun;
+    },
```

Comment:
This function is unnecessary just use dryRun directly.

Looks like this was useful in tests. Instead just create another tree walker with a different dryRun value.